activemodel/test/cases/errors_test.rb

```diff
@@ -189,6 +189,33 @@ def test_has_no_key
     assert !person.errors.added?(:name, "cannot be blank")
   end
 
+  test "add_once adds a specific message to an attribute only once" do
+    person = Person.new
+    message = "has only one error message"
+    person.errors.add_once(:name, message)
+    person.errors.add_once(:name, message)
+    assert_equal [message], person.errors[:name]
+    assert_not_equal [message, message], person.errors[:name]
```

Comment:
I think the `assert_not_equal` can be removed, since it is already checked with the `assert_equal`. What do you think?